elastic_app_search/client.py

```diff
@@ -289,6 +289,46 @@ def click(self, engine_name, options):
         endpoint = "engines/{}/click".format(engine_name)
         return self.session.request_ignore_response('post', endpoint, json=options)
 
+    def show_settings(self, engine_name):
```

Comment:
can you rename this to `get_search_settings` to stay consistent with the API Spec in the php client? https://github.com/elastic/app-search-php/blob/master/resources/api/api-spec.yml#L471

```suggestion
    def get_search_settings(self, engine_name):
```